ledger-core/v2/virtual/integration/vstatereport_if_pulse_changed_test.go

```diff
@@ -0,0 +1,92 @@
+// Copyright 2020 Insolar Network Ltd.
```

Comment:
Rename that file to simple `vstatereport_test.go`, i don't think that 1000 files (1 file  with `_test.go` suffix for each testcase) is a potentially good idea

Reply:
I think what we need better name for this test because we have vstatereport in handlers too. Maybe name must show what this message show SMObject